python_modules/dagster/dagster/core/core_tests/test_evaluator.py

```diff
@@ -237,16 +230,23 @@ def test_nested_missing_and_not_defined():
     ).reason == DagsterEvaluationErrorReason.FIELD_NOT_DEFINED
 
 
-MultiLevelDictType = build_config_dict_type(
-    ['TestMultiLevel'],
+MultiLevelDictType = types.Dict(
     {
-        'level_one_string_field': types.Field(types.String),
-        'level_two_dict': {
-            'level_two_int_field': types.Field(types.Int),
-            'level_three_dict': {
-                'level_three_string': types.Field(types.String),
-            },
-        },
+        'level_one_string_field':
+        types.Field(types.String),
+        'level_two_dict':
+        types.Field(
```

Comment:
we might consider additional sugar here, so that we can write `'level_two_dict': {...}`